### ISSUE TITLE
Organization/units - top right buttons are too close to each other, focus border not visible

### ISSUE THREAD

**Issue by kacperknapik** [SAP/spartacus#9527]
### Current
Top right buttons are too close to each other. When using tabbing through them, the focus state border is hiding underneath the other buttons.

EDIT: Added screenshots and url

To reproduce, go to `http://../powertools-spa/en/USD/organization/units` (for credentials, ask on priv).

![image](https://user-images.githubusercontent.com/22354613/97886006-5fe9fe80-1d28-11eb-8e00-2c8485793c9e.png)

### Expected
Move buttons further from each other, so the focus state border is visible correctly.

**Comment by wangzixi-diablo** [comment]
# More Background

See the focus state border: it intersects with other buttons when tab button is pressed:

![image](https://user-images.githubusercontent.com/58975336/97834181-26a69400-1d12-11eb-87b0-099042699191.png)

**Comment by wangzixi-diablo** [comment]
![image](https://user-images.githubusercontent.com/58975336/97841624-3083c380-1d21-11eb-80ac-e470c48d78a4.png)
